HousingSearchApi/data/elasticsearch/tenureIndex.json

```diff
@@ -14,120 +11,99 @@
         }
       },
       "paymentReference": {
-        "type": "keyword",
-        "normalizer": "my_normalizer",
-        "fields": {
-          "keyword": {
-            "type": "keyword",
-            "ignore_above": 256
-          }
-        }
+        "type": "keyword"
       },
       "startOfTenureDate": {
         "type": "date"
       },
       "endOfTenureDate": {
-        "type": "text",
-        "fields": {
-          "keyword": {
-            "type": "keyword",
-            "ignore_above": 256
-          }
-        }
+        "type": "keyword"
       },
       "householdMembers": {
```

Comment:
Ideally this would be `type: "nested` but that would break Temporary Accommodation and be a bunch of extra work